EveCM/ClientApp/app/reducers/adminPermissionsReducer.ts

```diff
@@ -17,7 +17,30 @@ export function roleInformation(state = adminPermissionsState.roleInformation, a
     switch (action.type) {
 
         case AdminPermissionsKeys.LOAD_ROLE_INFORMATION_SUCCESS:
-            return action.roleInformation;
+            return {
+                ...state,
+                data: action.roleInformation
+            };
+
+        case AdminPermissionsKeys.REMOVE_MEMBER_FROM_ROLE_SUCCESS: {
+            if (state.data.name == action.roleName) {
+                return Object.assign({ ...state }, {
+                    data: {
+                        ...state.data,
+                        users: [...state.data.users].filter(u => u.id !== action.user.id)
```

Comment:
You do not have to use the spread operator here. [].filter() returns a new array